model/patch_lifecycle.go

```diff
@@ -452,6 +452,17 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 		return nil, errors.Wrap(err, "fetching patch parameters")
 	}
 
+	authorEmail := p.GitInfo.Email
+	if p.GithubPatchData.AuthorUID != 0 {
+		u, err := user.FindByGithubUID(p.GithubPatchData.AuthorUID)
+		grip.Error(message.WrapError(err, message.Fields{
```

Comment:
I'd rather not log this if we aren't going to do anything with it; it's not really an error i think that a user might not have included this (although I see we use this same format elsewhere; we may just want to remove them both, or at least change them to info)

Reply:
Done just removed the error here.